src/static/css/main.less

```diff
@@ -81,6 +81,16 @@ body {
     overflow-wrap: break-word;
 }
 
+blockquote {
+
+  border-left: 0.25em solid @gray;
+  color: @dark-gray;
+  padding: 0.5em 1em 0.25em 1em;
+
+  &_p {
+    display: inline;
+  }
```

Comment:
Hmm. I think that's only useful if we decide to place a background image on the blockquote style or something. Not needed for now, and I'll remove it.